base_custom_info/models/custom_info_value.py

```diff
@@ -0,0 +1,304 @@
+                    ("relation", "=", "custom.info.template"),
+                    ("model_id.transient", "=", False),
+                    "!",
+                    ("model", "=like", "custom.info.%"),
+                ]
+            )
+            .mapped("model_id")
+        )
+        models = models.search([("id", "in", models.ids)], order="name")
+        return [
+            (m.model, m.name)
+            for m in models
+            if m.model in self.env and self.env[m.model]._auto
+        ]
+
+    @api.depends("property_id.field_type")
+    def _compute_field_name(self):
+        """Get the technical name where the real typed value is stored."""
+        for s in self:
+            s.field_name = "value_{!s}".format(s.property_id.field_type)
+
+    @api.depends("res_id", "model")
+    def _compute_owner_id(self):
+        """Get the id from the linked record."""
+        for record in self:
+            record.owner_id = "{},{}".format(record.model, record.res_id)
+
+    def _inverse_owner_id(self):
+        """Store the owner according to the model and ID."""
+        for record in self.filtered("owner_id"):
+            record.model = record.owner_id._name
+            record.res_id = record.owner_id.id
+
+    @api.depends(
+        "property_id.field_type",
+        "field_name",
+        "value_str",
+        "value_int",
+        "value_float",
+        "value_bool",
+        "value_id",
+    )
+    def _compute_value(self):
+        """Get the value as a string, from the original field."""
+        for s in self:
+            if s.field_type == "id":
+                s.value = s.value_id.display_name
+            elif s.field_type == "bool":
+                s.value = _("Yes") if s.value_bool else _("No")
+            else:
+                s.value = getattr(s, s.field_name, False)
+
+    def _inverse_value(self):
+        """Write the value correctly converted in the typed field."""
+        for record in self:
+            if (
+                record.field_type == "id"
+                and record.value == record.value_id.display_name
+            ):
+                # Avoid another search that can return a different value
+                continue
+            record[record.field_name] = self._transform_value(
+                record.value, record.field_type, record.property_id,
+            )
+
+    @api.constrains("property_id", "value_str", "value_int", "value_float")
+    def _check_min_max_limits(self):
+        """Ensure value falls inside the property's stablished limits."""
+        for record in self:
+            minimum, maximum = record.property_id.minimum, record.property_id.maximum
+            if minimum <= maximum:
+                value = record[record.field_name]
+                if not value:
+                    # This is a job for :meth:`.~_check_required`
+                    continue
+                if record.field_type == "str":
+                    number = len(self.value_str)
```

Comment:
```suggestion
                    number = len(record.value_str)
```

Reply:
> 
> 
> This should fix it

@kittiu fixed, could you check?

Reply:
Thank you. Can you also check the travis please.